root/static/scripts/edit/URLCleanup.js

```diff
@@ -1540,6 +1540,13 @@ const CLEANUPS = {
       return /^https:\/\/musopen\.org\/music\/\d+\/$/.test(url);
     },
   },
+  'muziekweb': {
+    match: [new RegExp('^(https?://)?www\\.muziekweb\\.eu/Link/', 'i')]
+    type: LINK_TYPES.otherdatabases,
+    clean: function (url) {
+      return url.replace(/^(?:https?:\/\/)?(?:www\.)?muziekweb\.(eu|nl)\/(en\/)?Link\/([0-9A-Z]+).*$/, 'https://www.muziekweb.eu/Link/$3/');
```

Comment:
It works but some improvements can be made:
* Skip capturing groups which are not used in the replacement pattern by adding `?:`
* Optionally match any language code in case the website gets more translation languages.
```suggestion
      return url.replace(/^(?:https?:\/\/)?(?:www\.)?muziekweb\.(?:eu|nl)\/(?:[a-z]{2}\/)?Link\/([0-9A-Z]+).*$/, 'https://www.muziekweb.eu/Link/$1/');
```